agent/ui_endpoint.go

```diff
@@ -439,7 +439,7 @@ func prepSummaryOutput(summaries map[structs.ServiceName]*ServiceSummary, exclud
 				sum.ChecksCritical++
 			}
 		}
-		if excludeSidecars && sum.Kind != structs.ServiceKindTypical {
+		if excludeSidecars && !(sum.Kind == structs.ServiceKindTypical || sum.Kind == structs.ServiceKindIngressGateway) {
```

Comment:
```suggestion
		if excludeSidecars && sum.Kind != structs.ServiceKindTypical && sum.Kind != structs.ServiceKindIngressGateway {
```
I feel like this would be less confusing exploded out, possibly with newlines after the `&&`

Reply:
Variable terminology-wise it is weird to think of an ingress gateway as a "sidecar".